Naloga213/report/report.tex

```diff
@@ -0,0 +1,164 @@
+formule. Privzeli smo, da sta vozlišči simetrični glede na koordinatno izhodišče, torej $x_2 = -x_1$. 
+Za lažji zapis smo označili $x_1 = x$, iz česar sledi $x_2 = -x$. Sistem enačb \eqref{eq:1}-\eqref{eq:4} se je tako poenostavil na tri 
+enačbe
+\begin{align*}
+    A + B &= 2 \\
+    Ax - Bx &= 0 \\
+    Ax^2 + Bx^2 &= \frac{2}{3},
+\end{align*}
+zadnja enačba pa služi preverjanju pravilnosti rešitve.
+
+Iz prve enačbe smo izrazili $B$:
+\[
+B = 2 - A
+\]
+To smo vstavili v drugo enačbo, $Ax - Bx = 0$:
+\[
+Ax - (2-A)x = 0
+\]
+Izpostavili smo $x$:
+\[
+x(A - (2-A)) = 0 \implies x(2A - 2) = 0 \implies 2x(A-1) = 0
+\]
+Iz te enačbe sledita dve možni rešitvi: $x=0$ ali $A=1$.
+Primer $x=0$ bi pomenil, da je $x_1=x_2=0$. Če to vstavimo v tretjo enačbo, 
+dobimo $A(0)^2 + B(0)^2 = 0$, kar je v protislovju z zahtevo, da je rezultat 
+$\frac{2}{3}$. Zato smo to možnost zavrgli.
+
+Edina preostala možnost je $A=1$. Iz zveze $B = 2 - A$ takoj sledi:
+\[
+B = 2 - 1 = 1
+\]
+Uteži $A=1$ in $B=1$ vstavimo v tretjo enačbo, da določimo še vrednost vozlišč:
+\[
+x^2 + x^2 = \frac{2}{3} \implies 2x^2 = \frac{2}{3} \implies x^2 = \frac{1}{3}
+\]
+Rešitev je $x = \pm\frac{1}{\sqrt{3}}$. S tem smo dobili vozlišči:
+\[
+x_1 = -\frac{1}{\sqrt{3}}, \quad x_2 = \frac{1}{\sqrt{3}}
+\]
+Za konec smo rešitev preverili še v prvotni četrti enačbi:
+\[ Ax_1^3 + Bx_2^3 = \left(-\frac{1}{\sqrt{3}}\right)^3 + \left(\frac{1}{\sqrt{3}}\right)^3 = 
+-\left(\frac{1}{\sqrt{3}}\right)^3 + \left(\frac{1}{\sqrt{3}}\right)^3 = 0 \]
+Ker je tudi ta enačba izpolnjena, smo potrdili pravilnost izpeljane rešitve.
+
+\section{Transformacija na poljuben interval}
+
+Ko imamo izpeljano pravilo na standardnem intervalu $t \in [-1, 1]$, ga moramo preslikati 
+na poljuben interval $x \in [a, b]$. To storimo z linearno transformacijo, ki 
+preslika $t \mapsto x$:
+\[
+x(t) = m t + c
+\]
+Konstanti $m$ in $c$ določimo iz pogojev, da se krajišča intervalov preslikajo vase:
+\[
+x(-1) = -m + c = a \quad \text{in} \quad x(1) = m + c = b
+\]
+Z reševanjem sistema dveh enačb dobimo $c = \frac{a+b}{2}$ in $m = \frac{b-a}{2}$. 
+Transformacija je torej:
+\[
+x(t) = \frac{b-a}{2}t + \frac{a+b}{2}
+\]
+Za substitucijo v integralu potrebujemo še zvezo med diferencialoma. Z odvajanjem zgornje 
+enačbe po $t$ dobimo:
+\[
+\frac{dx}{dt} = \frac{b-a}{2} \implies dx = \frac{b-a}{2} dt
+\]
+Originalni integral lahko sedaj zapišemo kot integral po spremenljivki $t$ na intervalu $[-1, 1]$:
+\[
+\int_{a}^{b} f(x) \,dx = \int_{-1}^{1} f\left(\frac{b-a}{2}t + \frac{a+b}{2}\right) \frac{b-a}{2} \,dt
+=  \frac{b-a}{2} \int_{-1}^{1} f\left(\frac{b-a}{2}t + \frac{a+b}{2}\right) \,dt
+\]
+Z uporabo izpeljanega Gauss-Legendrovega integracijskega pravila sedaj pridemo do končne formule:
+\[
+\int_{a}^{b} f(x) \,dx \approx \frac{b-a}{2} \left[ f\left( \frac{a+b}{2} - \frac{b-a}{2\sqrt{3}} \right) 
++ f\left( \frac{a+b}{2} + \frac{b-a}{2\sqrt{3}} \right) \right]
+\]
+
```

Comment:
Manjka izpeljava ocene za napako. Napako dobite tako, da v formulo vstavite polinom $x^4$ (za $x^3$ je formula točna).